src/main/java/dinkplugin/LevelNotifier.java

```diff
@@ -79,15 +82,36 @@ public void attemptNotify() {
         plugin.messageHandler.createMessage(plugin.config.levelSendImage(), body);
     }
 
-    public void handleLevelUp(String skill, int level) {
+    public void handleLevelUp(String skill, int level, int xp) {
         if (plugin.isSpeedrunWorld()) return;
-        if (plugin.config.notifyLevel() && checkLevelInterval(level) && currentLevels.get(skill) != null) {
-            if (level == currentLevels.get(skill)) {
-                return;
+
+        int virtualLevel = level < 99 ? level : getLevelForExperience(xp);
```

Comment:
It's technically possible to add a toggleable setting for this but: on toggle, one would need to call `#reset` or we'd need to keep a map of xp per skill to be able to switch between virtual and non-virtual levels

let me know if this should be implemented in this PR

Reply:
I imagine that if you have levelups enabled, you wouldn't mind sending a notification on virtual levelups